route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManager.cs

```diff
@@ -6833,10 +6833,37 @@ public TerritoryZone UpdateTerritory(TerritoryZoneParameters territoryParameters
 
                                 try
                                 {
-                                    var streamTask = ((StreamContent) response.Result.Content).ReadAsStreamAsync();
-                                    streamTask.Wait();
-
-                                    errorResponse = streamTask.Result.ReadObject<ErrorResponse>();
+                                    errorResponse = default; 
```

Comment:
is it possible to add unit test to see the problem? Like having a red unit test without a fix, and green unit test with fix?